### PR TITLE
TDP - 23 Add Posterization Shader with Dithering

### PR DESCRIPTION
**Pull request by ASlightlyOvergrownCactus**
Don't worry about the crazy amount of changed files, most of them are just the palette images.

Adds a shader within a test scene that posterizes the entire screen to be a set of colors from a given palette, with dithering from a given bayer matrix. This is meant to be an option for a route of art style that we could take if we want to ^.^, but no worries if not.

### CODE REVIEW

File: resources/Shaders/TestImages/mainmenu.png
Comment (funkychicken493) [inline resources/Shaders/TestImages/mainmenu.png:1]: mainmenu.png needs to be removed we do not have rights to this image

**Review by funkychicken493 — CHANGES_REQUESTED** [review]

File: resources/Shaders/TestImages/mainmenu.png
Comment (ASlightlyOvergrownCactus) [reply resources/Shaders/TestImages/mainmenu.png:1]: ughhhh booooooring

**Review by funkychicken493 — CHANGES_REQUESTED** [review]
Delete the .godot folder and merge from main

**Comment by ASlightlyOvergrownCactus** [comment]
Ok, .godot folder is fixed and ready to merge

**Review by funkychicken493 — APPROVED** [review]
LGTM

**Review by AhmadAJabr — APPROVED** [review]
Approved

**Pull request merged**